plugins/command-line/prism-command-line.js

```diff
@@ -54,6 +55,26 @@ Prism.hooks.add('complete', function (env) {
 	prompt.className = 'command-line-prompt';
 	prompt.innerHTML = lines;
 
+	var filterContent = pre.getAttribute('data-filter', '');
+	if (filterContent.length > 0) {
+		for (var i = 0; i < content.length; i++) {
+			var line = content[i];
+			if (line.slice(0, promptText.length) == promptText) {
+				// We have a command -- strip off the prompt from the source text and wrap in <span>
+				content[i] = '<span class="command-line-command">' + line.slice(promptText.length + 1) + 
+					'</span>';
```

Comment:
If I understand this correctly, it means users who don't use the `data-filter` attribute won't benefit from the styled `<span>`. Am I right? Couldn't this behaviour be added in every cases?

Reply:
I did not want to cause any possibly breaking change, hence the clear separation. I am more than willing to rework it if desired.

Reply:
I think that would be better to have a predictible behaviour when the span is always present, indeed. But yes we need to make sure nothing breaks.